frontend/src/components/core/Block/Block.tsx

```diff
@@ -129,11 +139,13 @@ class Block extends PureComponent<Props> {
   private renderBlock(
     element: BlockElement,
     index: number,
-    width: number
+    width: number,
+    deltaBlock: Delta.IBlock
   ): ReactNode {
+    const { collapsible, label, collapsed } = deltaBlock
     return (
       <div key={index} className="stBlock" style={{ width }}>
-        <Block
+        <this.WithCollapsibleBlock
```

Comment:
I guess maybe withCollapsible is not the right name since it doesn't necessary become collapsable. There's a check within the HOC to determine if it is collapsable. Do you think it's more of a naming thing or we need a separate switch?

Reply:
Mostly, there doesn't seem to be a clean place to add on the column type (e.g. would a column be another wrapper here? would that be handled inside the withCollapsible.tsx?)

Reply:
Updated but I'm thinking for columns, we could just append to the optionalProps.